tests/migrations/downgrade_test.py

```diff
@@ -132,3 +169,283 @@ def get_migrate_version(db_engine: Engine):
             return connection.execute(
                 text("SELECT version FROM migrate_version")
             ).scalar()
+
+    def test_downgrade_from_v23_6(
+        self, alembic_runner: MigrationContext, caplog: LogCaptureFixture
+    ):
```

Comment:
- Added unit tests for extra caution when executing the downgrade migration script from version 23.6.